sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -60,7 +60,7 @@ def __init__(
             RDS security group.
         vpc : ec2.Vpc
             VPC into which to put the resources that require networking.
-        sqs_queue: sqs.Queue
+        sqs_queues: sqs.Queue
```

Comment:
Should this be this now?
```suggestion
        sqs_queues: list[sqs.Queue]
```